docs/modules/opa/pages/usage-guide/OpenTelemetry.adoc

```diff
@@ -0,0 +1,30 @@
+= OpenTelemetry
+:description: Ship OPA traces and logs to OpenTelemetry
+:opa-docs: https://v1-4-2--opa-docs.netlify.app/configuration/#distributed-tracing
+
+Opa supports sending OpenTelemetry traces as stated in {opa-docs}[the documentation].
+
+As at SDP 25.7, `configOverrides` are not supported.
```

Comment:
Does this mean that we never supported it but will do in the future (as is suggested below, and which seems to be the case from the changelog), or that we are temporarily not supporting it (but _will_ do again at some point)?

Reply:
Then I would make this a little clearer with e.g. something like

> SDP 25.7 still does not support `configOverrides`, but this will be addressed in a future release.

or:

> As of SDP 25.7, `configOverrides` are (still) not supported.

Reply:
f816d4d